lib/main.dart

```diff
@@ -66,8 +66,8 @@ class _MyHomePageState extends State<MyHomePage> with TickerProviderStateMixin {
     LocationService location = new LocationService();
     try {
       currentLocation = await location.getLocation;
-    } on PlatformException {
-      print(MissingPluginException);
+    } on PlatformException catch(e){
```

Comment:
dartfmt

Reply:
what's wrong with my format?

Reply:
it's not formatted :)
`catch(e){`

Reply:
all right! committing this right away! ;)